lib/util/player_configuration.js

```diff
@@ -433,6 +433,7 @@ shaka.util.PlayerConfiguration = class {
       },
       forceTransmux: false,
       insertFakeEncryptionInInit: true,
+      correctEc3Enca: false,
```

Comment:
Since you mention in the description that it is required on Chromecast, I think this should be enabled by default on Chromecast.

```suggestion
      correctEc3Enca: shaka.util.Platform.isChromecast(),
```

Reply:
My main hesitation with that is this bug/behavior is very dependent on the streams, and I'm not sure it's ideal to automatically opt users into a behavior that will do extra processing for every init segment if they don't necessarily need it with their streams.